pkgs/by-name/pr/proton-ge-custom/package.nix

```diff
@@ -0,0 +1,53 @@
+{ lib
+, stdenvNoCC
+, fetchurl
+, writeScript
+}:
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "proton-ge-bin";
```

Comment:
Note that what I'm mostly concerned about is the attribute name, not the pname.

You have to move the file to change that in `pkgs/by-name/`.